src/providers/colors/ColorProvider.hpp

```diff
@@ -13,6 +13,7 @@ enum class ColorType {
     Whisper,
     RedeemedHighlight,
     FirstMessageHighlight,
+    SelfMessage,
```

Comment:
Same thing here, not sure if the name is super nice. A comment at the very least would be good